lib/active_model/serializer/adapter/json_api.rb

```diff
@@ -78,6 +87,28 @@ def serializable_hash(options = nil)
           hash
         end
 
+        # TODO: look into caching
+        # rubocop:disable Style/AsciiComments
+        # definition:
+        #   ☑ toplevel_errors array (required)
+        #   ☐ toplevel_meta
+        #   ☐ toplevel_jsonapi
+        # rubocop:enable Style/AsciiComments
```

Comment:
@buren I've documented the spec in a way in the code here and in the Error module.  ref discussion in https://github.com/rails-api/active_model_serializers/pull/1004/files#r52708419

I think adding a link here would be appropriate `{http://jsonapi.org/format/#errors JSON API Errors}`